### ISSUE TITLE
Edge browser UA and ban

### ISSUE THREAD

**Issue by AMTNate** [sealemar/BingRewards#347]
Is it possible to get banned for using an Edge browser UA? And should I be using a different UA for each account?

**Comment by specu** [comment]
they dont provide reason for a ban so nobody will be able to tell you for sure. In my opinion if it's a UA of an actual browser then it's not possible. possible reasons are: finding cookies of another account in your browser, reusing phone numbers, even within household, or redeeming too frequently from 1 IP address.

**Comment by AMTNate** [comment]
So if I have three accounts in the script, and the script is set to go 3 times a day, at different times each day in case of error, do you think that would be detectable?

**Comment by specu** [comment]
In my opinion, based on this info - no.

**Comment by AMTNate** [comment]
Ok thanks.

**Comment by aliasfoxkde** [comment]
AMTNate, I don't think so, you're allowed 5 accounts per household in the Terms of Service.. meaning you can run 5 accounts from 1 IP address (which is the external one hitting the web). So there's no way they can tell. I run 5 accounts from one IP and redeem all with the same household phone number. I haven't been banned yet.. been doing it for over a year (until the script stopped working with the new dashboard that is). Thanks guys!

**Comment by specu** [comment]
@aliasfoxkde the script should be working with the new dashboard now.

**Comment by aliasfoxkde** [comment]
Yeah, I know.. that's why I've been following. I appreciate it and I'll test it when I get home today!

**Comment by specu** [comment]
good feedback with the phones though. perhaps I redeemed too frequently in 1 day on those accounts from that 1 phone number. I was under the impression that each account needs to have a different phone.

**Comment by aliasfoxkde** [comment]
I've played around with it a little and had extra dummy accounts. Microsoft seems more lenient if you redeem using phone and I guess it depends on how much you redeem because validation is where they get you. I tried using a FreedomPop phone to scale rewards, damn things might be "free" but they're just VOIP. So they became kid phones. I don't watch my rewards, I redeem no more than once a month and they're staggered (not the same day). Maybe that helps

**Comment by aliasfoxkde** [comment]
If I can ask, why doesn't the script do Quizzes? It really wouldn't be that difficult to do or is there something I'm missing? Also saw that kinkeadtech has their own bot that solves quizzes, including the drag and drop ones. Also I use a pattern to solve them in the least amount of moves when I do, I don't even read the questions. Thanks

**Comment by specu** [comment]
@aliasfoxkde nobody had time to focus on solving quizzes, it takes a bit more than other activities.

**Comment by S3phi40T** [comment]
@specu phones are confirmed. You can use the same phone to activate/unlock your M$ account.
It's the case to use different phone for activating rewards (first redeem).

We have been testing it with my friend.
I was using VPN + multiple regions = delayed ban.
He had one account where he used to collect points from multiple regions = ban.
He had 2 accounts activated with the same phone - ban.

Generally, any email that suspicious action has been detected on your account = ban.
If you receive message to contact support if you want to redeem = ban.

https://github.com/sealemar/BingRewards/issues/344#issuecomment-362771663
At current state, I have each account (more than 5) activated with different physical land line and they are working fine.

Quizzes where you click the answer are easy with Chrome + SessionBox. There is bug, so you can answer 1st question 3 (or 5) times and you get rewarded for full quiz. Only those drag and drop ones have to be run regular way. I open them in Edge as browser can save account info and you can swap accounts quickly.

**Comment by aliasfoxkde** [comment]
Great, thanks for the info @S3phi40T and @specu. I still haven't had a chance to test the updated code but I will.

I did run some bots from a VPN for a while, but I have a PureVPN lifetime subscription and they no longer have US based locations. I've tried using a US based VPS as well, it worked for the most part. Now I just run bots from my NAS/server in a FreeBSD Jail. I live in California. I usually don't mess with them graphically version of rewards unless redeeming and I use my phone. Rarely will I do quizzes. I'd be curious to give scripting the quizes a shot and I love python so it should be a good downtime task. Obviously I can't promise anything but I'd like to contribute.

Sorry if I turn this into more a forum with going off topic.

I might try using the two bots in conjunction on a VM (kathawala/RewardsBot and sealemar/BingRewards). And I'll try out the Quiz bug you mention @S3phi40T
